llvm/test/CodeGen/PowerPC/aix-tls-gd-double.ll

```diff
@@ -156,10 +156,11 @@ define void @storesTIInit(double %Val) #0 {
 ; SMALL32:       # %bb.0: # %entry
```

Comment:
These cases(with `-gd-` in their names) are seems trying to test global-dynamic mode, so maybe it makes more sense we keep the original test points and add new runtime for the local-dynamic mode.

Reply:
The change is subtle, since `TIInit` is `internal`, all changes in this file is about `TIInit`.

Reply:
Looks like there are some overlapping test coverage in these TLS related cases. Maybe after enable local-dynamic, we could go through a process to do some cleanup or move some cases to their own file.